app/controllers/movies_controller.rb

```diff
@@ -7,7 +7,12 @@ def index
   end
 
   # GET /movies/1 or /movies/1.json
-  def show; end
+  def show
+    @seances = Seance
+               .where(movie_id: params[:id])
+               .where(start_time: DateTime.current + 30.minutes..)
```

Comment:
You are missing something after `30.minutes..` this line returns an error
Also you can use a helper to simplify this
```suggestion
               .where(start_time: 30.minutes.from_now..<Whatever is missing here>)
```

Reply:
Thanks for all the comments! 
The line with `30.minutes..` did not return an error, it returned everything greater than this condition, but I changed it anyway since I'll only need records for 7 days.